backend/tests/test_auth_api.py

```
@@ -2,160 +1,70 @@
+        phone="1234567890",
+        role="customer",
+        is_verified=True
+    )
+    db.add(user)
+    db.commit()
+    db.refresh(user)
+    return user
+
+
+@pytest.fixture
+def override_get_db(test_db):
```

Ruff (0.8.2)

Missing return type annotation for public function `override_get_db`

Add return type annotation: `None`

(ANN201)